html/dom/interfaces.sub.https.html

```diff
@@ -200,7 +200,7 @@ <h1>HTML IDL tests</h1>
       PeerConnection: [],
       MediaStreamEvent: [],
       ErrorEvent: [],
-      WebSocket: ['new WebSocket("wss://foo")'],
+      WebSocket: ['new WebSocket("wss://{{hosts[][nonexistent]}}")'],
```

Comment:
Since I had to ask;
https://web-platform-tests.org/writing-tests/server-features.html?highlight=nonexistent#tests-involving-multiple-origins

As you mentioned, optionally we could keep the filename the same and plumb the host reference into an external helper JS file. Could be worthwhile here, for such a huge number of subtests.